src/index.html

```diff
@@ -60,7 +63,8 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
                 </div>
 
                 <p>
-                    Coming from another platform? Docs: <a href="/docs/get-started/flutter-for/ios-devs">iOS</a>,
+                    Coming from another platform? Docs:
```

Comment:
`Docs:`
Sentence fragment

Reply:
Yeah, again, UX Designer and Martin.  :)